practice-app/app/location.py

```diff
@@ -0,0 +1,72 @@
+from .views import app, session, Base
+from flask import render_template, request
+from flask_login import login_required, current_user
+from sqlalchemy import Column, String, ForeignKey, Float, Integer
+from sqlalchemy.orm import Mapped, mapped_column
+import os
+import googlemaps
```

Comment:
Can you add this package into requirements.txt

Reply:
I added